provider_shopper/lib/screens/cart.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:provider_shopper/models/cart.dart';
+
+class MyCart extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Cart', style: Theme.of(context).textTheme.display4),
+        backgroundColor: Colors.white,
+      ),
+      body: Container(
+        color: Colors.yellow,
+        child: Column(
+          children: [
+            Expanded(
+              child: Padding(
+                padding: const EdgeInsets.all(32),
+                child: _CartList(),
+              ),
+            ),
+            Container(height: 4, color: Colors.black),
+            _CartTotal()
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _CartList extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    var textTheme = Theme.of(context).textTheme.title;
+    var cart = Provider.of<CartModel>(context);
+
+    return ListView(
```

Comment:
Partially at @xster's urging, I've started using ListView.builder everywhere possible, since it's more efficient. You're never going to have a zillion items in this list, but it'd still be a good thing to show.

Also (and this is a bit of a nit), consider using a ListTile ratehr than just a Text and sticking the bullet character in the `leading`  property.